accounts/models.py

```
@@ -2,23 +2,27 @@
 from enum import unique
 from time import timezone
 from django.db import models
-from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin,
+)
 import uuid
 from django.utils import timezone
```

Ruff (0.12.2)

Redefinition of unused `timezone` from line 3

(F811)